modules/nw-running-dpdk-rootless.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/using-dpdk-and-rdma.adoc
+
+:_content-type: CONCEPT
+[id="nw-running-dpdk-rootless_{context}"]
+= A test rootless DPDK pod template
+
+The following `dpdk-rootless-container` pod demonstrates how to build a pod spec for a rootless DPDK application.
+
+.An example `dpdk-rootless-container` pod
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dpdk-app
+spec:
+  containers:
+  - name: dpdk-rootless-container
+    image: <DPDK_image> <1>
+    securityContext:
+      capabilities:
+        drop: ["ALL"]<2>
+        add:<3>
+          - IPC_LOCK
+          - SYS_RESOURCE
+          # - NET_RAW #for mlx only
+          - NET_ADMIN
+      fsGroup: 801 <4>
+      runAsUser: 1001 <5>
+      runAsGroup: 1001 <6>
+      privileged: false <7>
+      allowPrivilegeEscalation: true <8>
+      runAsNonRoot: true <9>
+    volumeMounts:
+    - mountPath: /dev/hugepages <10>
+      name: hugepage
+    resources:
+      limits:
+        hugepages-1Gi: "4Gi"  <11>
+        memory: "2Gi"
+        cpu: "4" <12>
+    env:
+      - name: "RUN_TYPE"
+        value: "dpdk-application"
+    volumeMounts:
+    - mountPath: /mnt/huge
+      name: hugepages
+  volumes:
+    - name: hugepages
+      emptyDir:
+        medium: HugePages
+----
+<1> Specify the DPDK image which includes your application and the DPDK library used by application.
+<2> Dropping all capabilities (`ALL`) from the container's securityContext means that the container will not have any special privileges beyond what is necessary for normal operation.
+<3> Specify additional capabilities required by the application inside the container for hugepage allocation, system resource allocation, and network interface access the Virtual Function I/O (VFIO) device.
```

Comment:
Can we add additional info about Mellanox. Mlx card explicitly require NET_RAW capability.